packages/pcset/index.d.ts

```diff
@@ -0,0 +1,127 @@
+type Binary = "0" | "1";
+type Chroma = Binary[12];
```

Comment:
Maybe this is a better option (Chroma is a 12-length string with 0s and 1s... but thats not _typeable_, isn't it?):
```suggestion
type Chroma = string;
type ChromaArray = Binary[12];
```

Reply:
Yeah my bad, I realized Binary[12] does not mean what I thought it meant 😅